essential_reading/00_StartHere.md

```diff
@@ -0,0 +1,49 @@
+# Foundations
```

Comment:
I'm not 100% set on this...would like to chat more once you read through all my comments. But I'm kind of picturing one background document with resources that we can link to from other places. Maybe a notebook is explaining a concept and we say "go check out resources x, y, and z from our resources page (link) if you are not familiar with them before beginning this tutorial." Then the tutorials we are building will really be focused on preprocessing datasets or evaluating models. If there's a need for tutorials that we develop that don't fit in those categories, let's talk about what they are and why we should be the owners of that content.

Reply:
And I hope this doesn't feel critical of the plan you were putting together! This is all great stuff! I'm just trying to help think through our scope on this team and consider how our work should fit in with our tasks in HyTEST or other groups at USGS. All the stuff in here is critical learning. I just want to outsource as much content ownership as possible so our maintenance doesn't get unmanageable.

Reply:
I don't perceive sharp criticism.  Frankly, I am glad to have an overseer with a vision to question and make decisions over the direction.  If a lot of this doesn't have a place, then that frees up hours to focus on other stuff.  I'm not terribly attached. 

We should for-sure talk via slack or teams, tho.  Would probably save a lot of back-and forth.